### ISSUE TITLE
v0.9.56 Always Fails Due to Missing Config File

### ISSUE THREAD

**Issue by cwfitzgerald** [nextest-rs/nextest#926]
Seems to have a bug wrt config files in the latest release. If your repo doesn't have one, you get:

```
error: failed to parse nextest config at `/Users/cwfitzgerald/programming/wgpu/.config/nextest.toml`

Caused by:
  No such file or directory (os error 2)
```

**Comment by sunshowers** [comment]
Whoops! Fixing.

**Comment by sunshowers** [comment]
Just released 0.9.57 with a fix. In the meanwhile I'm yanking 0.9.56.

**Comment by sunshowers** [comment]
Builds will be up in around 20-30 minutes.

**Comment by sunshowers** [comment]
OK, also yanked from `/latest`.

Need to do a post-mortem, yanking should have taken under 5 minutes but it actually took closer to 20. Apologies again.

**Comment by marc2332** [comment]
Thank you ! ⚡ 🐎 🏎

**Comment by thomasmost** [comment]
Thank you!

**Comment by sunshowers** [comment]
Post-mortem issue is #927.

**Comment by sunshowers** [comment]
Reopening because of https://github.com/taiki-e/install-action/issues/178.

**Comment by sunshowers** [comment]
The issue has been addressed, taiki-e/install-action now downloads 0.9.57.
